c2corg_ui/static/js/feed.js

```diff
@@ -138,6 +336,60 @@ app.FeedController.prototype.getDocumentsFromFeed = function() {
   }.bind(this));
 };
 
+/**
+ * get latest topics
+ * @private
+ */
+app.FeedController.prototype.getLatestTopics_ = function() {
+  this.busyForum = true;
+  this.api.readLatestForum().then(function(response) {
+    this.handleForum(response);
+  }.bind(this), function() { // Error msg is shown in the api service
+    this.busyForum = false;
+    this.errorForum = true;
+  }.bind(this));
+};
+
+/**
+ * number cannot be < 0 and cannot be decimal
+ * @param {number} n
+ * @private
+ */
+app.FeedController.prototype.naturalNumber = function(n) {
+  if (n < 0) {
+    return 0;
+  } else {
+    if (n > 10) {
+      return 10;
+    } else {
+      return Math.round(n);
+    }
+  }
+};
+
+/**
+ * simulate size for a doc
+ * @param {Object} doc
+ @return {number}
+ * @public
+ */
+app.FeedController.prototype.sizeEstimator = function(doc) {
```

Comment:
It's a good practise to use an action verb as name of function, eg. ``estimateSize``.
In addition do you think this function is robust enough? If the layout of the cards is changed (in the CSS or partials), all the size calculations below will have to be updated by hand as well, won't they?

Reply:
fixed